app/src/app/routes/Router.tsx

```diff
@@ -36,17 +38,30 @@ const Router = (): JSX.Element => {
             <CreateMapping />
           </Route>
           <Route exact path="/sources/:sourceId/mappings/:mappingId">
+            <MappingHeader />
             <Mapping />
           </Route>
           <Route
             exact
             path="/sources/:sourceId/mappings/:mappingId/attributes/:attributeId"
           >
+            <MappingHeader />
             <Mapping />
           </Route>
           <Route exact path="/sources/:sourceId/mappings/:mappingId/edit">
             <EditMapping />
           </Route>
+          <Route exact path="/sources/:sourceId/mappings/:mappingId/preview">
+            <MappingHeader />
```

Comment:
so do we keep this header ?

Reply:
I think for now ? That's what I understood ...

Reply:
because I moved it